vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import tsconfigPaths from 'vite-tsconfig-paths';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [tsconfigPaths()],
```

Comment:
this plugin is used to fix utilize the 'paths' configuration in the tsconfig files.